components/BottomNavigation/src/MDCBottomNavigationBar.h

```diff
@@ -164,6 +164,11 @@ typedef NS_ENUM(NSInteger, MDCBottomNavigationBarAlignment) {
  */
 @property(nonatomic, assign) CGFloat itemsContentHorizontalMargin;
 
+/**
+ The amount of horizontal padding on the leading/trailing edges of each bar item. Defaults to 12.
+ */
+@property(nonatomic, assign) CGFloat itemsHorizontalPadding;
```

Comment:
rename: itemHorizontalPadding

Since this is the padding between the items it is odd to pluralize this property. The plural implies it is the right and left padding of them all (to me)

Reply:
actually scratch that. looks like you are following other properties like: itemsContentHorizontalMargin 
I guess its item's content horizontal margin.

Reply:
The reason I suggested insets is because what it does is it takes the whole width of the bottom navigation bar, divides it by the number of items to get an initial width, and then it insets the actual frame of the item by this property value on both sides. Does that make sense? So if the value is 12 (that's the default) there will actually be 24 points of horizontal space between the individual items. I'm fine with leaving it how it's been though. They were calling it padding when it was private.